### PR TITLE
LPS-162811 Check that the dropItem has a parent form

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/drag-and-drop/useDragAndDrop.js [hunk 1]
```diff
@@ -386,16 +387,24 @@ function computeDrop({dispatch, layoutDataRef, onDragEnd, state}) {
...
-		else if (state.dropItem.isWidget) {
+		else if (
+			state.dropItem.isWidget &&
+			hasFormParent(state.dropItem, layoutDataRef.current)
+		) {
 			message = Liferay.Language.get(
 				'widgets-cannot-be-placed-inside-a-form-container'
 			);
 		}
+		else if (state.dropItem.parentId !== state.dropTargetItem.itemId) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/drag-and-drop/useDragAndDrop.js:398]: Shouldn't this be just `else {`? In which case does this happen?

**Pull request closed without merging**